src/lib/resource/interceptor/manager.es

```diff
@@ -0,0 +1,148 @@
+/**
```

Comment:
You should be using the one moved to syn-core right?
I suppose these changes are pending... Please answer.